backend/canisters/user/impl/src/model/communities.rs

```diff
@@ -28,7 +28,7 @@ impl Communities {
     }
 
     pub fn any_updated(&self, since: TimestampMillis) -> bool {
-        self.communities.values().any(|c| c.last_updated() > since)
+        self.communities.values().any(|c| c.last_updated() > since) || self.removed.iter().any(|g| g.timestamp > since)
```

Comment:
The latest one will always be last so you can do this -
`self.removed.last().map(|g| g.timestamp > since)`